website/src/test-utils/TestProviders.tsx

```diff
@@ -0,0 +1,23 @@
+import { RouterContext } from "next/dist/shared/lib/router-context";
+import { NextRouter } from "next/router";
+import { SessionProvider } from "next-auth/react";
+import { PropsWithChildren } from "react";
+
+import { createMockRouter } from "./createMockRouter";
+
+const mockRouter = createMockRouter({});
+// TODO: better session mock, should be improved with upcoming tests
+const mockSession = {};
+// the i18n is initialized in `jest.setup.js`, i18next does not have a clean way
+// of adding a provider context
+export const TestProviders = ({
+  router,
+  session,
+  children,
+}: PropsWithChildren<{ router?: NextRouter; session?: any }> = {}) => {
+  return (
+    <RouterContext.Provider value={router ?? mockRouter}>
```

Comment:
Maybe we can add the `FlagsProvider` to this too?

Reply:
Yeah, I did not need it for the PR so I left it. I hope that is okay. maybe in an upcoming PR?

We should definitely push harder on testing.

Reply:
Yeah, I think it's fine for now. We don't depend on feature flags so much.